test/actions/projects.spec.js

```diff
@@ -0,0 +1,11 @@
+import { getProjects } from '../../lib/actions/projects';
+
+describe('getProjects', () => {
+  it('should be a function', () => {
+    expect(typeof getProjects).toBe('function');
+  });
+
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 

```suggestion
  it('should create an action', () => {
```